pkgs/by-name/ne/nerdfetch/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gitUpdater
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nerdfetch";
+  version = "8.1.1";
+
+  src = fetchFromGitHub {
+    owner = "ThatOneCalculator";
+    repo = "NerdFetch";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-mSRS7MUgaa14mgPYkbC4O1/gXxbyDKOiVCRgoh8He/c=";
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/bin
+      cp $src/nerdfetch $out/bin
+  '';
+
+  passthru.updateScript = gitUpdater {
+    rev-prefix = "v";
+  };
+
+  meta = with lib;{
+    description = "A POSIX *nix (Linux, macOS, Android, *BSD, etc) fetch script using Nerdfonts";
+    homepage = "https://codeberg.org/thatonecalculator/nerdfetch";
```

```suggestion
    homepage = "https://github.com/ThatOneCalculator/NerdFetch";
```

The codeberg repo is just a mirror which is not up to date.